index.js

```diff
@@ -74,7 +74,7 @@ module.exports = function (options, cb) {
       return taskRunner.runPromisified(params);
     })
     .then((taskDefinition) => {
-      if (taskDefinition.failures) {
+      if (taskDefinition.failures && taskDefinition.failures.length) {
```

Comment:
It's an array and it's always present?

Reply:
actually, shouldn't we check that length > 0?

Reply:
That's exactly what we're doing! `0` is falsy. I can switch if you like but I think it's more idiomatic as written.

Reply:
Of course, I golf in js so grain of salt wrt my opinions of what's legible lol

Reply:
I think `> 0` is clearer as to the intent, even if it's redundant.